openfisca_us/data/datasets/cps/raw_cps.py

```diff
@@ -0,0 +1,156 @@
+from io import BytesIO
+from zipfile import ZipFile
+from openfisca_tools.data import PublicDataset
+import pandas as pd
+import requests
+from tqdm import tqdm
+from openfisca_us.data.storage import OPENFISCA_US_MICRODATA_FOLDER
+
+
+class RawCPS(PublicDataset):
+    name = "raw_cps"
+    label = "Raw CPS"
+    folder_path = OPENFISCA_US_MICRODATA_FOLDER
+
+    url_by_year = {2020: "https://some_url.org"}
```

Comment:
?

Reply:
Removed, but will add the GitHub h5 release URL when it's up